### PR TITLE
Overload UriHelper to forceLoad the page (imported from Blazor PR 1154)

### PR DESCRIPTION
**Pull request by SteveSandersonMS**
Imported from https://github.com/aspnet/Blazor/pull/1154

### CODE REVIEW

**Comment by SteveSandersonMS** [comment]
@rynowak Another one I've already reviewed and consider ready, but need another approval before it can be merged. Please feel free to review further if you want!

**Review by rynowak — APPROVED** [review]

**Pull request merged**
